src/main/java/uk/gov/companieshouse/document/generator/consumer/document/service/impl/MessageServiceImpl.java

```
@@ -123,24 +126,34 @@ private Map<String, Object> setCompletedDebugMap(DocumentGenerationCompleted com
         Map<String, Object> completedParams = new HashMap<>();
         completedParams.put(DESCRIPTION_IDENTIFIER, completed.getDescriptionIdentifier());
         completedParams.put(DESCRIPTION, completed.getDescription());
-        completedParams.put(COMPLETED_DOCUMENT, completed);
+        completedParams.put(COMPLETED_DOCUMENT, getObjectMapper(String.valueOf(completed)));
 
         return completedParams;
     }
 
     private Map<String, Object> setFailedDebugMap(DocumentGenerationFailed failed) {
 
         Map<String, Object> failedParams = new HashMap<>();
-        failedParams.put(FAILED_DOCUMENT, failed);
+        failedParams.put(FAILED_DOCUMENT, getObjectMapper(String.valueOf(failed)));
 
         return failedParams;
     }
 
-    private Map<String, Object> setStartedDebugMap(DocumentGenerationStarted started) {
-
+    private Map<String, Object> setStartedDebugMap(DocumentGenerationStarted started)  {
         Map<String, Object> startedParams = new HashMap<>();
-        startedParams.put(STARTED_DOCUMENT, started);
+        startedParams.put(STARTED_DOCUMENT, getObjectMapper(String.valueOf(started)));
 
         return startedParams;
     }
+
+    private JsonNode getObjectMapper(String logStatus){
```

This method should be named something more appropriate like getLogStatusTree
The catch block shouldn't be empty, do something with the exception or let it crash
And setting node = null is rough, what happens when the catch block let's the null fall through? I'd assume a vague hard to debug NPE, at the very least add a null check

---

made changes